alacritty_terminal/Cargo.toml

```diff
@@ -0,0 +1,95 @@
+[package]
+name = "alacritty_terminal"
+version = "0.3.2"
+authors = ["Joe Wilm <joe@jwilm.com>"]
+license = "Apache-2.0"
+build = "build.rs"
+description = "GPU-accelerated terminal emulator - library for embedders"
+readme = "../README.md"
+homepage = "https://github.com/jwilm/alacritty"
+edition = "2018"
+
+[dependencies]
+libc = "0.2"
+notify = "4"
+bitflags = "1"
+font = { path = "../font" }
+errno = "0.2"
+parking_lot = "0.7"
+serde = "1"
+serde_derive = "1"
+serde_json = "1"
+serde_yaml = "0.8"
+vte = "0.3"
+mio = "0.6"
+mio-extras = "2"
+copypasta = { path = "../copypasta" }
+xdg = "2"
+log = "0.4"
+clap = "2"
+fnv = "1"
+unicode-width = "0.1"
+glutin = { version = "0.21.0", features = ["icon_loading"] }
+base64 = "0.10.0"
+static_assertions = "0.3.0"
+terminfo = "0.6.1"
+url = "1.7.1"
+crossbeam-channel = "0.3.8"
+
+[target.'cfg(unix)'.dependencies]
+nix = "0.13"
+signal-hook = { version = "0.1", features = ["mio-support"] }
+
+[target.'cfg(any(target_os = "linux", target_os = "freebsd", target_os="dragonfly", target_os="openbsd"))'.dependencies]
+x11-dl = "2"
+
+[target.'cfg(windows)'.dependencies]
+winpty = { path = "../winpty" }
+mio-named-pipes = "0.1"
+miow = "0.3"
+dunce = "1.0"
+winapi = { version = "0.3.7", features = ["impl-default", "winuser", "synchapi", "roerrorapi", "winerror", "wincon", "wincontypes"]}
+dirs = "1.0"
+widestring = "0.4"
+mio-anonymous-pipes = "0.1"
+image = "0.21.0"
+
+[target.'cfg(target_os = "macos")'.dependencies]
+objc = "0.2.2"
+
+[features]
+default = []
+# Enabling this feature makes shaders automatically reload when changed
+live-shader-reload = []
+nightly = []
+bench = []
+
+[build-dependencies]
+gl_generator = "0.11.0"
+
+[target.'cfg(windows)'.build-dependencies]
+embed-resource = "1.1.4"
+tempfile = "3.0.4"
+reqwest = "0.9"
+zip = "0.5"
+
+[package.metadata.deb]
+maintainer = "Joe Wilm <joe@jwilm.com>"
+license-file = ["LICENSE-APACHE", "3"]
+extended-description = """\
+Alacritty is the fastest terminal emulator in existence. Using the GPU for \
+rendering enables optimizations that simply aren't possible without it.  """
+depends = "$auto"
+section = "rust"
+priority = "optional"
+assets = [
+    ["target/release/alacritty", "usr/bin/", "755"],
+    ["extra/linux/alacritty.desktop", "usr/share/applications/", "644"],
+    ["extra/logo/alacritty-term.svg", "usr/share/pixmaps/Alacritty.svg", "644"],
+    ["extra/completions/alacritty.bash", "usr/share/bash-completion/completions/alacritty", "644"],
+    ["extra/completions/alacritty.fish", "usr/share/fish/completions/alacritty.fish", "644"],
+    ["extra/completions/_alacritty", "usr/share/zsh/vendor-completions/_alacritty", "644"],
+    ["extra/alacritty.info", "usr/share/terminfo/a/alacritty", "644"],
+]
+maintainer-scripts = "extra/linux/debian"
```

Comment:
My Cargo.toml has this moved in the `alacritty` subcrate. I've actually made sure that debian still builds with my fork by test-building it once so it's probably a good idea to follow its lead in this regard.

Since we can only create debs from binaries, this isn't useful here.

Reply:
Done